ehrql/utils/sdc_utils.py

```diff
@@ -0,0 +1,17 @@
+"""Statistical Disclosure Control (SDC) utilities.
```

Comment:
This is more of an aesthetic preference than anything particularly concrete, but I'm not sure I'd break this out into a utils module like this. Utils _generally_ speaking (although we're definitely not fully consistent here) generally contain code which isn't "core business logic" and either needs to be called from multiple places, or is just too long-winded and faffy to include in the module which needs it.

I wonder if a better structure would be to pull the code out of the `calculate` module and have a `ehrql.measures.disclosure_control` module which contains that code plus the stuff here in utils.

Reply:
That makes sense, Dave. Thanks for describing your rationale. I have added `ehrql.measures.disclosure_control`.

I originally wanted to keep `apply_sdc_to_measure_results` next to `get_measure_results` to make it clear that the former returns a tuple that matches the structure of the latter (we're using the tuple as a record, not just as an immutable list, in other words). However, a separate module keeps everything related to disclosure control together, which is clearer.

I guess that this might suggest we need a `namedtuple` to make the record's structure explicit. I'm going to leave that question for another day.